### PR TITLE
[Improvement] Draft UG for v1 and v2, Add some FAQ, Add backbone of command summary

### LINKED ISSUE #195: Update User Guide for List Schools

### LINKED ISSUE #196: Update User Guide for Obtain Contacts

### PR DESCRIPTION
**Pull request by louisjoety**
Fixes #195 
Fixes #196

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -160,22 +160,117 @@ Expected Output:
 Please provide a valid index of the course plan you would like to delete.
 ```
 
+### List out all the possible schools from the options: `list schools`
+List out all the schools users could possibly go for their SEP
+
+Format: `list schools`
```
Comment (chewycj) [inline docs/UserGuide.md:166]: Detailed documentation of the list schools command!

**Review by chewycj — APPROVED** [review]
LGTM!! 👍 Good start for the QnA section too!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -160,22 +160,117 @@ Expected Output:
...
+-----------------------------------------------------
+The University of Melbourne
+The Australian National University
+Victoria University of Wellington
+The University of Western Australia
+-----------------------------------------------------
+```
+
+### Obtain contacts from the list of universities `obtain`
+Delete a course mapping plan that was initially saved into the Personal Tracker.
```
Comment (chewycj) [inline docs/UserGuide.md:179]: I think this part was not updated for the overview for the `obtain` command!

Comment (louisjoety) [reply docs/UserGuide.md]: Okay, ammended! @chewycj

**Pull request merged**